src/polar/PolarAngleAxis.tsx

```diff
@@ -145,7 +147,7 @@ export class PolarAngleAxis extends PureComponent<Props> {
 
       return (
         <Layer
-          className="recharts-polar-angle-axis-tick"
+          className={clsx('recharts-polar-angle-axis-tick', get(tick, 'className'))}
```

Comment:
Why do we need `get` here?

Reply:
@ckifer 
Thank you for the review!
This is because of type errors.
Should it be implemented with type guards?
![image](https://github.com/recharts/recharts/assets/9676316/69eb86e0-089f-4d9b-8272-fd6604711e79)

Reply:
Yeah I'd rather use type guards than lodash where possible. A quic type guard utility for className should suffice

Reply:
@ckifer 
Updated!